ankara-universitesi-hukuk-fakultesi.csl

```diff
@@ -0,0 +1,1492 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" delimiter-precedes-et-al="never" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="1" page-range-format="expanded" default-locale="tr-TR">
+  <info>
+    <title>Ankara Üniversitesi - Sosyal Bilimler Enstitüsü - Hukuk Fakültesi (Türkçe)</title>
+    <title-short>AUHF</title-short>
+    <id>http://www.zotero.org/styles/ankara-universitesi-hukuk-fakultesi</id>
+    <link href="http://www.zotero.org/styles/ankara-universitesi-hukuk-fakultesi" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <link href="http://www.zotero.org/styles/ankara-universitesi-hukuk-fakultesi" xml:lang="tr" rel="documentation"/>
+    <!-- Türkçe'de küçük i büyük İ, ingilizce'de küçük i büyük I olduğu için, Türkçe word kullanılıyorsa
+      son aşamada i İ düzeltmesi yapmak gerekiyor (dipnot ve kaynakça'ya özel find/replace) -->
+    <author>
+      <name>Av. Yavuz Selim TAŞKIN</name>
+      <email>yst@yst.tr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <summary xml:lang="tr">Ankara Üniversitesi Sosyal Bilimler Enstitüsü - Hukuk Fakültesi Atıf Usulü</summary>
+    <updated>2024-11-12T18:49:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="tr">
```

Comment:
please remove any terms from these locale overrides that are already included in the locale (https://github.com/citation-style-language/locales/blob/master/locales-tr-TR.xml)
We'd be grateful if you could actually prepare a PR with any missing translations in that locale.